components/org.wso2.carbon.identity.conditional.auth.functions.user/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/user/TerminateUserSessionImpl.java

```diff
@@ -59,8 +59,8 @@ private boolean terminateUserSession(AuthenticatedUser authenticatedUser, String
         try {
             UserRealm userRealm = Utils.getUserRealm(tenantDomain);
             if (userRealm != null) {
-                String userId = UserSessionStore.getInstance()
-                        .getUserId(username, Utils.getTenantId(tenantDomain), userStoreDomain);
+                String userId = FrameworkUtils.resolveUserIdFromUsername(Utils.getTenantId(tenantDomain),
```

Comment:
Since this method is deprecated in the master, and authentication framework supports retrieving authenticated user's user id you can use `authenticatedUser.getUserId()`

Reply:
Updated the code.